test/integration_tests/test_models_correctness.py

```diff
@@ -0,0 +1,124 @@
+# Authors: Bruno Aristimunha
```

Comment:
can you run black on this file? it violates a number of pep8 rules

Reply:
also why a few folder in test? as you only test biot I would call this file test_biot.py 

i would have kept it in unit_tests/models/

Reply:
I was thinking of creating more tests about full integration in the future, but you are right. It is a topic for another PR.